mfr/extensions/utils.py

```diff
@@ -0,0 +1,22 @@
+from urllib.parse import urlparse, parse_qs, urlencode
+from mfr.extensions import settings
+
+
+def download_from_template(func):
+    """
+    Check if LOCAL_DEVELOPMENT and if so
+    Replace the docker domain with localhost (for use in local development).
```

Comment:
Copyedit:  "If MFR is being run in a local development environment (i.e. LOCAL_DEVELOPMENT is True), we need to replace the external host (the one the user provides, default: "localhost") with the internal host (the one the backend services communicate on, default: 192.168.168.167)."